CHANGELOG.md

```diff
@@ -55,6 +56,23 @@ Markdown Spec](https://github.github.com/gfm/).
   ```
   Not only is the response format itself different, but there may be fewer
   facets returned since normalized values which match are combined
+### Changed
+- upgraded to Rails 6.1.7 and Ruby 3
+- changes reflect new api schemas in Datura, which make heavy use of nested fields
+
+### Migration
+- in Datura repos config `private.yml` api to `"api_version": "2.0"` to take advantage of new bucket aggregation functionality (or `"api_version": "1.0"` for legacy repos that have not been updated for the new schema). Please note that a running API index can only use one ES index at a time, and each ES index is restricted to one version of the schema.
+- Use Elasticsearch 8.5 or later
+- If you are using ES with security enabled, you must configure credentials with Rails in the API repo. See https://guides.rubyonrails.org/v6.1/security.html. Configure the VSCode editor. Run `EDITOR="code --wait" rails credentials:edit` and add
+```
+elasticsearch:
+  user: username
+  password: *****
+```
+to the secrets file and then close the window to save. Do not commit `config/master.key` (it should be in `gitignore`)
+- Orchid apps that connect to the API should use `facet_limit` instead of `facet_num` in options.
+- Add nested facets as described above, if desired
```

Comment:
This part probably should point out `config/initializers/config.rb` needs the facet code we discussed in https://github.com/CDRH/prairie_forestry/pull/124 and https://github.com/CDRH/teaa/pull/110